backend/sendmessage/src/login-user.ts

```diff
@@ -18,7 +20,9 @@ const COHEN_SCALE = [
   '∞',
 ];
 
-async function loginUser(userId, groupId, config) {
+const EXPIRY_TIME_IN_HOUR = process.env.EXPIRY_TIME_IN_HOUR || '1';
```

Comment:
Here as well: when is this default needed?

Reply:
similar